experiments/AMIP/moist_mpi_earth/coupler_driver.jl

```diff
@@ -120,19 +129,20 @@ end
 
 # init coupler
 coupler_field_names = (:T_S, :z0m_S, :z0b_S, :ρ_sfc, :q_sfc, :albedo, :F_A, :F_E, :F_R, :P_liq)
-coupler_fields =
+coupler_fileds =
     NamedTuple{coupler_field_names}(ntuple(i -> ClimaCore.Fields.zeros(boundary_space), length(coupler_field_names)))
 model_sims = (atm = atmos_sim, ice = ice_sim, lnd = land_sim, ocn = ocean_sim)
 dates = (; date = [date], date0 = [date0], date1 = [date1])
 
-coupler_sim = CouplerSimulation(
```

Comment:
I think using `coupler_sim` is clearer!

Reply:
I agree. The timing macro would break though with `cs = coupler_sim` within the timing loop (i.e. `cs` needs to be in the global scope). Because of this behavior, we will need to revisit using this macro vs just functions, but removing it now seems like a major rework, so I put this on the TODO list after the AMIP showcase.